docs/operations/migration_policies.md

```diff
@@ -0,0 +1,182 @@
+# Migration Policies
+
+Migration policies provides a new way of applying migration configurations to Virtual Machines. The policies
+can refine Kubevirt CR's `MigrationConfiguration` that sets the cluster-wide migration configurations. This way,
+the cluster-wide settings serve as a default that can be refined (i.e. changed, removed or added) by the migration
+policy.
+
+Please bear in mind that migration policies are in version `v1alpha1`. This means that this API is not fully stable
+yet and that APIs may change in the future.
+
+## Overview
+
+KubeVirt supports [Live Migrations](./live_migration.md) of Virtual Machine workloads.
+Before migration policies were introduced, migration settings could be configurable only on the cluster-wide
+scope by editing [KubevirtCR's spec](https://kubevirt.io/api-reference/master/definitions.html#_v1_kubevirtspec)
+or more specifically [MigrationConfiguration](https://kubevirt.io/api-reference/master/definitions.html#_v1_migrationconfiguration)
+CRD.
+
+Several aspects (although not all) of migration behaviour that can be customized are:
+- Bandwidth
+- Auto-convergence
+- Post/Pre-copy
+- Max number of parallel migrations
+- Timeout
+
+Migration policies generalize the concept of defining migration configurations, so it would be
+possible to apply different configurations to specific groups of VMs.
+
+Such capability can be useful for a lot of different use cases on which there is a need to differentiate
+between different workloads. Differentiation of different configurations could be needed because different
+workloads are considered to be in different priorities, security segregation, workloads with different
+requirements, help to converge workloads which aren't migration-friendly, and many other reasons.
+
+## API Examples
+
+#### Migration Configurations
+Currently, the MigrationPolicy spec will only include the currently relevant configurations from KubevirtCR's
```

Comment:
```suggestion
Currently the MigrationPolicy spec will only include the relevant configurations from KubevirtCR's
```

Reply:
nit: Maybe elaborate about what the `relevant configurations` are?

Reply:
I changed it to:
```
Currently the MigrationPolicy spec will only include the following configurations from KubevirtCR's
MigrationConfiguration (in the future more configurations that aren't
part of Kubevirt CR are intended to be added):
```

By "relevant" I meant everything that is suitable for a use in a customizable scope (which is what migration policies are built for). For example, the number of maximum parallel migrations per node is not suitable to be determined by a migration policy.

But, in the near future we're planning to add configs that aren't suitable for a cluster-wide use, therefore would exist only for migration policies and not to Kubevirt CR.

I didn't want to get into all those details in the user-guide (I'm not sure the avarage user would be interested in that). What do you think about the new wording?